spring-boot-project/spring-boot-docs/src/docs/asciidoc/web/spring-security.adoc

```diff
@@ -223,7 +223,32 @@ Again, the same properties are applicable for both servlet and reactive applicat
 
 Alternatively, you can define your own `OpaqueTokenIntrospector` bean for servlet applications or a `ReactiveOpaqueTokenIntrospector` for reactive applications.
 
+To enable audience validation, set the `spring.security.oauth2.resourceserver.jwt.audiences` property in your Spring Boot application 
+configuration file. This property specifies the expected value(s) of the aud claim in JWTs.
```

Comment:
Instead of "aud claim" let's be more explicit here "audience claim". Similarly below as well. `aud` is not really a common acronym.

Reply:
> Instead of "aud claim" let's be more explicit here "audience claim". Similarly below as well. `aud` is not really a common acronym.

@wilkinsona do you agree with @at758 ? I have seen `aud` and `iss` used frequently in the code.

Reply:
@bikash30851 I don't. What you have written is fine. It aligns with the terms used in [Spring Security's documentation](https://docs.spring.io/spring-security/reference/servlet/oauth2/resource-server/jwt.html) where you'll find both `aud` and `iss`.